Views/Settings/Settings.swift

```diff
@@ -216,7 +216,8 @@ struct Settings: View {
         Section("Misc".localized) {
             Button("Feedback".localized) { UIApplication.shared.open(URL(string: "mailto:feedback@kiwix.org")!) }
             Button("Rate the App".localized) {
-                let url = URL(string: "itms-apps://itunes.apple.com/us/app/kiwix/id997079563?action=write-review")!
```

Comment:
@BPerlakiH Please replace the App store ID using `Brand.appstoreId`

Reply:
@tvision251 I am not sure if I understand your point here. 
With these changes the value from Brand.appStoreId is used.